src/components/ProductTable.jsx

```diff
@@ -10,15 +10,31 @@ const PRODUCTS = [
   { name: 'Mango', price: '$2.00', stocked: false }
 ];
 
-const ProductTable = () => {
-  // useState hook replaces need for call to setState()
-  let [inStockOnly, setInStockOnly] = useState(false);
-  
+const ProductTable = (props) => {
+  // QUESTION: is it best practice to initialize state within a child component, if that child component is the only place it will be used (i.e. other components don't need to know about it/its current value?)
+  // or is it best practice to initialize state as high up the hierarchy/as globally as possible (i.e. ProductPage) and pass it down as far as it needs to go (in this case, just down one level to this file)?
+  // second option seems to defeat the purpose of creating ProductPage in the first place (other than to handle count state)?
```

Comment:
You want to move it up to the top level that needs to know that information and no higher. If you find yourself managing the state of a component and passing it through a few different components (say 3+), that's called prop drilling and its a sign that you might need a more sophisticated state management solution like useContext to avoid having to pass all the props through.